### PR TITLE
Classifier: Refactor types.union with references

### PR DESCRIPTION
**Pull request by eatyourgreens**
Refactor types.union for readbility, particularly where a model uses references to multiple types.
See https://github.com/mobxjs/mobx-state-tree/issues/514

Package:
lib-classifier

# Review Checklist

## General

- [x] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/src/store/WorkflowStepStore.js [hunk 1]
```diff
@@ -2,9 +2,6 @@ import { autorun } from 'mobx'
 import { addDisposer, getRoot, isValidReference, types } from 'mobx-state-tree'
 
 import Step from './Step'
-import taskRegistry, { taskModels } from '@plugins/tasks'
-
-const taskTypes = types.union(...taskModels)
```
Comment (wgranger) [inline packages/lib-classifier/src/store/WorkflowStepStore.js]: Is this simply cruft that was left in this component?

**Comment by eatyourgreens** [comment]
I think #1550 fixed the storybook. I'll need to rebase this branch and fix the merge conflict.

File: packages/lib-classifier/src/store/WorkflowStepStore.js (hunk 1 above)
Comment (eatyourgreens) [reply packages/lib-classifier/src/store/WorkflowStepStore.js]: I think so.

**Comment by wgranger** [comment]
This should be good. It was an error that I was erroneously running `yarn start` instead of `yarn dev`. I'll give it another look

**Comment by wgranger** [comment]
Looking better after running `yarn dev`, but I'll wait until the rebase to run storybook and test again.

**Comment by eatyourgreens** [comment]
Storybook should be fixed now.

**Comment by wgranger** [comment]
LGTM

**Pull request merged**